security/cmd/istio_ca/main.go

```diff
@@ -199,6 +205,8 @@ func init() {
 			selfSignedCAOrgDefault))
```

Comment:
Please remove this field and use identityDomain as the org. Currently `org` in X509 cert is a place holder not being used.

Reply:
do you mean remove `self-signed-ca-org` CLI parameter and use identityDomain as the certificate's Org field?

Reply:
removed